src/python/pants/backend/python/subsystems/black.py

```diff
@@ -16,3 +16,5 @@ def register_options(cls, register):
     super().register_options(register)
     register('--config', advanced=True, type=file_option, fingerprint=True,
               help="Path to formatting tool's config file")
+    register('--check', advanced=True, type=bool, fingerprint=True,
```

Comment:
Long-term, there's an interesting question of direction here.

Historically, we had the `lint` goal, which is basically `fmt-without-formatting` - in the future, would we like:
1. A `lint` goal alongside the `fmt` goal?
2. A `fmt`-wide flag called something like `--check`
3. A per-formatter flag called `--check`

I suspect long-term we want 1 or 2, and we should work that out before we name the v2 console_rule `fmt`, but while this is experimental until then, this works :)